code/mob/living/carbon/human.dm

```diff
@@ -2208,8 +2208,20 @@
 		if (slot_l_store, slot_r_store)
 			if (I.w_class <= W_CLASS_SMALL && src.w_uniform)
 				return 1
-		if (slot_l_hand, slot_r_hand)
-			return 1
+		if (slot_l_hand)
+			if (src.limbs.l_arm)
+				if (!istype(src.limbs.l_arm, /obj/item/parts/human_parts/arm) && !istype(src.limbs.l_arm, /obj/item/parts/robot_parts/arm))
+					return 0
```

Comment:
Github won't let me add suggestions, but is good to replace `return 0/1` with `return FALSE/TRUE` where appropriate. Update old code and all that.